robolectric-resources/build.gradle

```diff
@@ -6,8 +6,8 @@ dependencies {
     // Compile dependencies
     compile "com.ximpleware:vtd-xml:2.11"
     compile "com.google.guava:guava:19.0"
-    provided "org.robolectric:android-all:6.0.0_r1-robolectric-0"
-    provided "com.intellij:annotations:12.0"
+    compileOnly "org.robolectric:android-all:6.0.0_r1-robolectric-0"
```

Comment:
See the last commit on https://github.com/robolectric/robolectric/tree/api_24-jdk7-compat, I've refactored the `AndroidSdk` stuff there.